install/kubernetes/helm/subcharts/tracing/values.yaml

```diff
@@ -22,12 +24,30 @@ jaeger:
       # - secretName: jaeger-tls
       #   hosts:
       #     - jaeger.local
-replicaCount: 1
```

Comment:
This is used in `deployment-jaeger.yaml`. Probably not relevant for in-memory config, so may be best to remove from the jaeger deployment aswell.